Code/Editor/ViewportTitleDlg.h

```diff
@@ -101,18 +99,13 @@ class CViewportTitleDlg
     QStringList m_customFOVPresets;
     QStringList m_customAspectRatioPresets;
 
-    float m_prevMoveSpeed;
+    float m_prevMoveSpeedScale;
 
     // Speed combobox/lineEdit settings
-    double m_minSpeed = 0.01;
-    double m_maxSpeed = 100.0;
-    double m_speedStep = 0.001;
-    int m_numDecimals = 3;
-
-    // Speed presets
-    float m_speedPresetValues[4] = { 0.01f, 0.1f, 1.0f, 10.0f };
-
-    double m_fieldWidthMultiplier = 1.8;
+    double m_speedScaleMin = 0.001;
+    double m_speedScaleMax = 100.0;
```

Comment:
Suggestion: Would be nice to allow a much higher max for very large levels (ex: 64 km x 64 km). :)

Reply:
Talked with Mike offline about this and tested and just updated 👍 (was 10.0, no 100.0)